lib/ebanq_api/profile.rb

```diff
@@ -28,12 +28,18 @@ def show(uid)
     #
     # ==== Attributes
     #
-    # * +uid+ (required) - The id of the user
+    # * +uid+ (required) - The id of the user (Number)
     # * +fields+ (required) - hash of params which needs to update,
     # e.g. {first_name: 'John', last_name: 'Doe'}
     def update(uid, fields)
       @client.make_request :post,
                            auth_path(uid), fields
     end
+
+    # Get list of allowed documents
+    # /api/v1/profile/
+    def list_of_allowed_documents
+      @client.make_request :post, PROFILE_API_PATH
+    end
```

Comment:
I'm not sure, but description looks like not for this function ('Get list of allowed document', but function used :post)

Reply:
It was a typo, fixed